authserver/api/static/css/style.css

```diff
@@ -1,51 +1,80 @@
-html,
-body {
-  height: 100%;
+:root {
+  --blue: #0fbaa8;
+  --light-blue: #0fbaa8b3;
+  --gray20: #333333;
```

Comment:
should set font-size: 100% on `html` so that it maintains the font size set by the user and `rem` will be based on it

Reply:
done:
https://github.com/brighthive/authserver/blob/e0dd4b2c3ca15c36f48e1af8562fdf0b772e14b3/authserver/api/static/css/style.css#L7-L15